deflect/qt/QmlStreamerImpl.cpp

```diff
@@ -341,6 +341,14 @@ void QmlStreamer::Impl::_onWheeled( double x_, double y_, double deltaY )
     QCoreApplication::postEvent( _quickWindow, e );
 }
 
+void QmlStreamer::Impl::_onPinched( double x_, double y_, QPointF delta )
+{
+    const QPoint point( x_ * width(), y_ * height( ));
+    QWheelEvent* e = new QWheelEvent( point, delta.y(), Qt::NoButton,
```

Comment:
pinch should not be translated to wheel I thought?!? QmlStreamer should or can handle gestures/touch points natively, no?